exercises/complex-numbers/src/example/kotlin/ComplexNumber.kt

```diff
@@ -0,0 +1,26 @@
+data class ComplexNumber(val real: Double = 0.0, val imag: Double = 0.0)
+
+operator fun ComplexNumber.plus(other: ComplexNumber)
```

Comment:
Does that work for operation functions? If so, yes! If not, I liked them outside so that the order of definition matched the order in which these methods were exercised by the tests. I'll have a play locally.